src/components/subscribe/subscribe-form.js

```diff
@@ -1,47 +1,20 @@
-import React, { useState } from "react";
+import React from "react";
 import PropTypes from "prop-types";
 
 import { Hero } from "../hero/hero";
 import { Input } from "../input/input";
-import { Button } from "../button/button";
 
 import "./subscribe-form.scss";
 
-import { validateEmail } from "./email-validator.service";
 import { Wording } from "./constants";
+import { Button } from "../button/button";
 
-export const SubscribeForm = ({
-  compact,
-  loading,
-  success,
-  errorMessage,
-  onSubmit
-}) => {
-  const [inputValue, setInputValue] = useState();
-  const [isValid, setIsValid] = useState();
-
-  const handleInputChange = ({ target: { value } }) => {
-    setInputValue(value);
-  };
-
-  const handleSubmit = () => {
-    const validationResult = validateEmail(inputValue);
-    setIsValid(validationResult);
-    const canSubmit = validationResult && typeof onSubmit === "function";
-
-    if (canSubmit) {
-      onSubmit({
-        email: inputValue
-      });
-    }
-  };
-
+export const SubscribeForm = ({ compact }) => {
   const SubscribeButton = () => (
-    <Button onClick={handleSubmit} disabled={loading || success}>
+    <Button onClick={() => void 0} inputType="submit">
```

Comment:
We can remove `onClick`, it's been removed from the component
```suggestion
    <Button inputType="submit">
```